lib/index.js

```diff
@@ -384,7 +384,7 @@ class KuzzleCluster {
         collection,
         filters
       },
-      serializedFilter = filter && JSON.stringify(filter) || 'none';
+      serializedFilter = filter ? JSON.stringify(filter) : 'none';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2583)